extensions/fnutils/init.lua

```diff
@@ -4,55 +4,130 @@
 
 local fnutils = {}
 
+local pairs,ipairs = pairs,ipairs
```

Comment:
I'm curious if this actually gets us anything? Isn't it just saving a quick lookup in local scope, before going to global scope?